.cirrus.yml

```diff
@@ -143,6 +143,27 @@ gating_task:
     on_failure:
         failed_branch_script: '$CIRRUS_WORKING_DIR/$SCRIPT_BASE/notice_branch_failure.sh'
 
+# This task checks to make sure that we can still build an rpm from the
+# source code using contrib/rpm/podman.spec.in
+rpmbuild_task:
+
+    only_if: $CIRRUS_BRANCH != $DEST_BRANCH
+    depends_on:
+        - "gating"
+    env:
+        CIRRUS_WORKING_DIR: "/usr/src/libpod"
+
+    container:
+        image: quay.io/libpod/rpmbuild:latest
```

Comment:
Probably should be ``quay.io/libpod/rpmbuild:$DEST_BRANCH``, otherwise it won't play nice with other branches (here)...they all step on "latest".  Let me check the build setup on quay as well...

Reply:
...okay, I manually added a tag for 'master' on quay.